### ISSUE TITLE
Mage Quest Bugged - Investigate the Alchemist Shop

### ISSUE THREAD

**Issue by saracyy** [Wall-core/Everlook-Bugtracker#209]
After using the Cantation, the Rift Spawn are neutral and not hostile.  They will only attack after you attack them first.

After killing the Rift Spawn they just die.  They are not stunned so they cannot be contained and the quest can't progress.

**Comment by TheRealTilon** [comment]
I agree, mobs should be hostile. You also need to kill them and "put" them in the boxes.
But in order to get the boxes "filled", you both need to:
1. Be close to the corpse when using the box.
2. Be in the Cantation area when using the box.

So basically, you can kite them to be in a safer position, BUT the corpse need to be near the location you were at when using the Cantation, otherwise you won't be able to use the box.

**Comment by saracyy** [comment]
Confirmed I am able to use the box when in the area
